examples/docker-start.js

```diff
@@ -0,0 +1,6 @@
+import containers from 'k6/x/docker/containers';
+
+export default function () {
+  const containerId = 'YOUR_CONTAINER_ID';
+  containers.startContainer(containerId, {attach: true});
```

Comment:
What would attaching even mean in this context? That we block until the container exits?

Reply:
This interface is wrong, I just remove it and keep containerId only

Reply:
✅